io/disk/softwareraid.py

```diff
@@ -86,6 +81,11 @@ def extensivetest(self):
         Extensive software raid options are run viz create, delete, assemble,
         create spares, remove and add drives
         """
+        self.disk = self.disk.split(" ")
+        self.sparedisk = self.disk.pop()
+        self.remadd = ''.join(self.disk[-1:])
+        self.disk_count = len(self.disk)
+        self.disk = ' '.join(self.disk)
```

Comment:
why this is required i think it having  same effect when it is in  setup and also it make sense that we can get all variable in test setup ?